tests/e2e/taqueria-plugin-flextesa.spec.ts

```diff
@@ -262,6 +262,39 @@ describe("E2E Testing for taqueria typechecker and simulator tasks of the tezos-
         expect(stderr).toContain('Unknown primitive hello.');
     })
 
+    test("Verify that the Taqueria CLI will pass 0x00 (not 0, which is in int, not a byte) to the simulator", async () => {
+        // 1. Copy contract from data folder to taqueria project folder
+        await exec(`cp e2e/data/byteSlice.tz ${taqueriaProjectPath}/artifacts`)
+
+        // 2. Run taq simulate
+        const {stdout, stderr} = await exec(`taq simulate byteSlice.tz '0x12a47ef2' --storage '0x00'`, {cwd: `./${taqueriaProjectPath}`})
+
+        expect(stdout).toContain('0xa47ef2')
+        expect(stderr).toBe('');
+    })
+
+    test("Verify that taqueria simulator task can understand Left and Right (part of input and storage value) correctly", async () => {
```

Comment:
Is this test just checking the contents of the smart contract? I think this is something we should do for our tests but I worry about what contracts are getting executed. I would like to talk about how we can cover this if needed. Otherwise we shouldn't be too concerned about the contents of compiled contracts since that isn't something we should be worried about since other people elsewhere work on that